docs/source/getting_started/distributed_training.rst

```diff
@@ -12,4 +12,72 @@ Training with multiple gpus is also available from the command line: :ref:`cli-t
 For details on distributed training we recommend the following pages:
 
 - `Pytorch Distributed Overview <https://pytorch.org/tutorials/beginner/dist_overview.html>`_
-- `Pytorch Lightning Multi-GPU Training <https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html>`_
\ No newline at end of file
+- `Pytorch Lightning Multi-GPU Training <https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html>`_
+
+
+There are different levels of synchronization for distributed training. One can
+for example just sync the gradients in the backpropagation step. But we can also
+sync special layers such as batch norm such that they get the statistics from all
+the batches across the GPUs. Some of the additional synchronization might 
+improve the final model accuracy at the cost of longer training due to the 
+communication overhead.
+
+We did some simple experiments we share here:
+
+Distributed Training Tests
+---------------------------
+
+- dataset = cifar10 
```

Comment:
Use `:` instead of `=` and capital letters, i.e.

- Dataset: Cifar10
- Batch size: 512
- Epochs: 100